### PR TITLE
Roach beer, Kaiser toxin

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request
watch out for the #6707 possible merge conflict!
adds two new drugs: Kakerlakenbier(roach beer) and Monarchenblut(kaiser beer)
adds a new toxin: kaiseraurum. Can be obtained by grinding Kaiser meat
Roach beer works "ghetto serotrotium", meaning that it's a drug that gives you a sanity boost(double if you are a vagabond!), but you lose sanity as a withdrawal. Easy to get addicted. Made out of:
2 water, 1 fuel, 1 beer, 1 diplopreptum
Monarchenblut works as a combat stim that also gives you a sanity boost(double if you are a vagabond) and makes roaches friendly towards you. Fairly easy to get addicted. ~~Holy shit it's overpowered.~~ Made out of:
1 fuhrerole, 1 kaiseraurum, 2 kakerlakenbier
kaiseraurum works as an upgraded fuhrerole: boosts your stats, makes you a romch frend, but it's easy to overdose on and the addiction is __pain__.
## Why It's Good For The Game
GHETTO SEROTROTIUM
GHETTO SEROTROTIUM
better beer!
vagabond supremacy!
~~The only reason Monarchenblut is balanced is that you need to kill a Kaiser for it~~
## Changelog
:cl:
add: Added a new toxin: kaiseraurum. Can be obtained by grinding kaiser meat.
add: Added a new drug: Kakerlakenbier. Works as a ghetto serotrotium, can be made out of water, fuel, diplopreptum and beer.
add: Added a new drug: Monarchenblut. Works as a ghetto serotrotium, can be made out of kakerlakenbier, fuhrerole and kaiseraurum.
/:cl:

### CODE REVIEW

**Comment by Kegdo** [comment]
Also it's possible to add a in_glass sprite for this, if OUR SPRITERS do it

**Comment by SirRichardFrancis** [comment]
Look why haven't tested their code.

**Comment by Kegdo** [comment]
> Look why haven't tested their code.

silence

File: code/modules/reagents/reagents/drugs.dm [hunk 1]
```diff
@@ -273,3 +273,84 @@
...
+	nerve_system_accumulations = 40
+	addiction_chance = 60
+	overdose = REAGENTS_OVERDOSE
+	addiction_threshold = 15
+	sanity_gain_ingest = 2
+	taste_tag = list(TASTE_SOUR,TASTE_BUBBLY)
+
+/datum/reagent/drug/roachbeer/affect_ingest(mob/living/carbon/M, alien, effect_multiplier) ////// checks user for having a vagabond perk,
+	var/perk_check = effect_multiplier
+	if(M.stats.getPerk(PERK_LOWBORN))														////// increases sanity_gain to 4 if true
```
Comment (TheShown911) [inline code/modules/reagents/reagents/drugs.dm:294]: Lowborn isn't the vagabond perk.

Comment (Kegdo) [reply code/modules/reagents/reagents/drugs.dm:294]: fucj

Comment (Kegdo) [reply code/modules/reagents/reagents/drugs.dm]: issue adressed

**Review by drexample — APPROVED** [review]

**Comment by Kegdo** [comment]
How the?

**Comment by Kegdo** [comment]
#6764

**Pull request closed without merging**